extensions/common/api/management-api-configuration/src/main/resources/mgmt-api-version.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
also the other version files should contain a list, for consistency (it will help a lot the openapi publish/verification processes)

Reply:
ah ok. sure, no problem.